config/routes.rb

```diff
@@ -891,6 +891,8 @@
 
     :ems_cloud                => {
       :get  => %w(
+        arbitration_profiles
```

Comment:
looks like we only ever GET arbitration_profiles and only ever POST arbitration_profiles_edit, am I right?